plugin/trino-exasol/pom.xml

```diff
@@ -107,7 +107,7 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>exasol-testcontainers</artifactId>
-            <version>7.1.1</version>
+            <version>7.1.2</version>
```

Comment:
test containers was also recently [released](https://github.com/testcontainers/testcontainers-java/releases/tag/1.20.4). And it contains some fixes for retrying docker images, so it can make CI less flaky. See https://github.com/testcontainers/testcontainers-java/pull/9474

Reply:
Testcontainers are latest